bin/bump-version

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+NEW_VERSION=$1
+
+if ! [[ $NEW_VERSION =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+  echo "Pass a version in the format \"x.y.z\" as the first argument to this script"
+  exit 1
+fi
+
+ROOT_DIR=$(cd "$(dirname "$0")/.."; pwd -P)
+
+cd "$ROOT_DIR"
+
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+if ! [[ $CURRENT_BRANCH =~ ^release/v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
Should we not check for `$CURRENT_BRANCH =~ ^release/v$NEW_VERSION`?

Reply:
Yes. You're right. I added the regular expression matching after doing the first pass. I will fix this.